martinLi/assignments/treesBFS/lc102/lc102.js

```diff
@@ -3,22 +3,39 @@
 // Given a binary tree, populate an array to represent its level-by-level traversal. You should populate the values of all nodes of each level from left to right in separate sub-arrays.
```

Comment:
Looking for algo prep work here, input/outputs, edges/corners, assumptions, high level overview